### ISSUE TITLE
Syntax highlighting of trailing comments backtrack to first slash of line

### ISSUE THREAD

**Issue by juliendorra** [davidedc/livecodelab#298]
Adding a trailing comment to a line with slashes, the purple color for comments backtrack and now start from the first slash in the line, instead of the double slash that start the comment.

(trailing comments are useful and can be better for readability if well used)

See captures below:

![capture d ecran 2017-10-03 a 17 21 38](https://user-images.githubusercontent.com/109677/31142144-b25f6cf8-a879-11e7-80b2-8c7a43df3aab.png)

![capture d ecran 2017-10-03 a 17 21 28](https://user-images.githubusercontent.com/109677/31142145-b2878314-a879-11e7-8729-0790c4be102e.png)

**Comment by rumblesan** [comment]
yup, it's something we've been aware of for a while, but just never gotten around to fixing. I'll try to have a dig into it this weekend. syntax highlighting could maybe do with some improvements again.

@juliendorra would be interesting to hear any feedback you have based on the workshops you've run recently

**Comment by rumblesan** [comment]
This now seems to be sorted in the most recent cut of LCL which will ideally get released soon. will close this when it's live. currently available at http://livecodelab.net/play2.5v3/
